rust/agama-migrate-wicked/src/reader.rs

```diff
@@ -59,16 +59,35 @@ pub async fn read(paths: Vec<String>) -> Result<Vec<Interface>, io::Error> {
 
 #[cfg(test)]
 mod tests {
+    use crate::interface::{
+        AdSelect, Bond, FailOverMac, LacpRate, PrimaryReselect, Slave, XmitHashPolicy,
+    };
```

Comment:
```suggestion
    use crate::interface::*;
```
Before we add a endless list here, I think just `*` is nicer.